src/gwt/src/org/rstudio/studio/client/workbench/views/console/consoleStyles.css

```diff
@@ -184,3 +184,89 @@
    font-style: italic;
    color: #888888;
 }
+
```

Comment:
might be nice to alphabetize these

Reply:
I also want to factor all these hyperlink specific styles in the `.hyperlink` package, it is starting to clutter `ConsoleResources`.